blocks/resources/resources.css

```diff
@@ -37,6 +37,24 @@ table.resources > thead > tr > th:nth-child(3){
     width :10%;
 }
 
+table.resources .resource-data:not(:first-child) .resource-link[href$=".pdf"]:before {
```

Comment:
is this still needed if with the css changes to the global styles to add the PDF?

Reply:
Yes, because it removes the PDF icon for the links in the 3rd column.